pkg/plugin/processor/builtin/impl/cohere/command_examples_test.go

```diff
@@ -33,7 +33,7 @@ func ExampleCommandProcessor() {
 		Summary: `Generate responses using Cohere's command model`,
 		Description: `
 This example demonstrates how to use the Cohere command processor to generate responses for a record's ` + "`.Payload.After`" + ` field.
-The processor sends the input text to the Cohere API and replaces it with the model's response.`,
+The processor sends the input text from the configured "request.body" to the Cohere API and stores the model's response into the configured "response.body"`,
```

Comment:
is this what we meant here?
```suggestion
The processor sends the input text from the configured "inputField" to the Cohere API and stores the model's response into the configured "outputField"`,
```

Reply:
The embed processor currently uses inputField/outputField for the same purpose as request.body/response.body in rerank and command processors. That's why it is like this.